### DISCUSSION TITLE
Clickable Stat card

### DISCUSSION THREAD

**Discussion by UK-17** [h2oai/wave#1449] [Q&A]
Is there a way I can a user redirect to an external page (hyperlink) when he clicks on my stat card.

<img width="1440" alt="Screenshot 2022-05-27 at 6 34 35 PM" src="https://user-images.githubusercontent.com/40884732/170704798-4bccdca2-4a30-4d50-8840-a42da79b1d4b.png">

For example in this page if someone clicks on one of the stat (Open Regression card) , I can redirect them to its Jira page.

**Comment by aalencar** [comment]
Try handling `q.args.<stat_card_name>` and then [redirect using the meta card ](https://wave.h2o.ai/docs/examples/meta-redirect/).

**Reply by mturoci** [reply]
Stat cards are not clickable as they do not have a `name` attribute.

**Comment by mturoci** [comment]
Hi @UK-17, the stat cards are not clickable as of today. The intention is to just display data and that's it. However, I think your request makes sense. Wdyt @lo5?

**Reply by UK-17** [reply]
Thanks, that helps.
